hr_language/models/hr_language.py

```diff
@@ -30,12 +29,36 @@ class hr_language(models.Model):
         tools.scan_languages(),
         string=u"Language", required=True)
     description = fields.Char(
-        string=u"Description", size=64, required=True)
+        string=u"Description", size=64, required=False)
     employee_id = fields.Many2one(
         'hr.employee', string=u"Employee", required=True)
     can_read = fields.Boolean(u"Read", default=True, oldname='read')
     can_write = fields.Boolean(u"Write", default=True, oldname='write')
     can_speak = fields.Boolean(u"Speak", default=True, oldname='speak')
+    read_rating = fields.Selection(
+        string='Reading',
+        selection=LANGUAGE_RATING,
+        required=True,
+        default='0',
```

Comment:
As this field is now required, and you have put default value, you should warn in the README that all the existing data will be considered as no proficiency.